src/WebHandlers.cpp

```diff
@@ -58,18 +58,15 @@ AsyncStaticWebHandler& AsyncStaticWebHandler::setCacheControl(const char* cache_
 }
 
 AsyncStaticWebHandler& AsyncStaticWebHandler::setLastModified(const char* last_modified){
-  _last_modified = String(last_modified);
+  _last_modified = last_modified;
   return *this;
 }
 
-AsyncStaticWebHandler& AsyncStaticWebHandler::setLastModified(struct tm* last_modified){
-  auto formatP = PSTR("%a, %d %b %Y %H:%M:%S %Z");
-  char format[strlen_P(formatP) + 1];
-  strcpy_P(format, formatP);
-
-  char result[30];
-  strftime(result, sizeof(result), format, last_modified);
-  return setLastModified((const char *)result);
+AsyncStaticWebHandler& AsyncStaticWebHandler::setLastModified(const std::tm* last_modified){
+  constexpr size_t buffsize = sizeof("Fri, 27 Jan 2023 15:50:27 GMT");    // a format for LM header
+  char result[buffsize];
```

Comment:
`buffersize + 1`

https://cplusplus.com/reference/ctime/strftime/

Reply:
No, it should account null terminator also
```    constexpr size_t buffsize = sizeof("Fri, 27 Jan 2023 15:50:27 GMT");                                                                                      
    std::cout<< "buffsize = " << buffsize << "\n";
```
gives `buffsize = 30`